app.js

```diff
@@ -1,7 +1,9 @@
 const express = require("express");
 const app = express();
 
-const { getTopics } = require("./controllers/topics.controller");
+const { getTopics, getEndpoints } = require("./controllers/topics.controller");
```

Comment:
I would probably have getEndpoints either directly in app or in it's own controller. It's not related to topics specifically.